mappings/net/minecraft/client/render/entity/TntMinecartEntityRenderer.mapping

```diff
@@ -1 +1,10 @@
 CLASS net/minecraft/class_957 net/minecraft/client/render/entity/TntMinecartEntityRenderer
+	METHOD method_23190 renderFlashingBlock (Lnet/minecraft/class_2680;Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;IZ)V
+		COMMENT Renders a given blockstate into the given buffers either normally or with a bright white overlay.
+		COMMENT Used for rendering primed TNT either standalone or as part of a TNT minecart.
+		COMMENT @param drawFlash If true, a white semi-transparent overlay is added to the block to indicate the flash.
```

Comment:
This should be a javadoc comment on the parameter itself (select the parameter in Enigma and add docs to that).

Reply:
Learning something new everyday :D